apps/system/locales/system.en-US.properties

```diff
@@ -473,6 +473,11 @@ payment-flow=Purchase
 
 # Remote Debugger Connection Dialog
 remoteDebuggerMessage=An incoming request to permit remote debugging connection was detected. Allow connection?
+remoteDebuggerPromptUSB=An incoming request to permit USB debugging connection was detected. Allow connection?
+remoteDebuggerPromptTCP=An incoming request to permit remote debugging connection was detected from {{host}}:{{port}}. Allow connection?
```

Comment:
I feel this is too verbose, maybe you could reword to something shorter like "Allow USB debugging request?" and "Allow remote debugging request from {{host}}:{{port}}?"